CHANGELOG.md

```diff
@@ -13,7 +13,8 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ### Changed
 
-- CAPA only change for new releases: render release version in config instead of cluster-aws version in App resource.
+- **BREAKING** CAPA only change for new releases: render release version in config instead of cluster-aws version in App resource.
```

Comment:
I have to admit I don't really understand the changelog entry. Can we explain this in more natural language? Also, which commands are affected?

Reply:
I have updated changelog so it's more clear.